framework/src/modules/token/types.ts

```diff
@@ -14,24 +14,25 @@
 
 import { MethodContext, ImmutableMethodContext } from '../../state_machine';
 import { CCMsg } from '../interoperability/types';
+import { JSONObject } from '../../types';
 
 export type TokenID = Buffer;
 
-export interface ModuleConfig {
-	minBalances: {
-		tokenID: string;
-		amount: string;
-	}[];
-	feeTokenID: string;
-	accountInitializationFee: string;
-	escrowInitializationFee: string;
-}
-
 export interface MinBalance {
 	tokenID: Buffer;
 	amount: bigint;
 }
 
+export interface ModuleConfig {
+	minBalances: MinBalance[];
+	ownChainID: Buffer;
```

Comment:
ownChainID does not exist in the module config (it comes from the genesis config)

Reply:
Yes, `ownChainID` is added to the module config during module init. The module needs to pass it to method, which uses it. Would it be better to remove it from `ModuleConfig` interface, and then merge `moduleConfig` object with an object that contains only `ownChainID` property, and pass that to `method.init()`?

Reply:
i think it would be confusing to have it in this interface because it looks like it's changeable from config, but it's not.
probably better to just have different interface for method

Reply:
but actually we have another issue to remove ownChainID, so it's fine to keep it as it is

Reply:
Moved `ownChainID` to `MethodConfig` interface.